.travis.yml

```diff
@@ -47,6 +52,9 @@ before_install:
 script:
   - npm install
   - npm test
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      docker run --rm -i --env SKIP_SASS_BINARY_DOWNLOAD_FOR_CI=true -v $DOCKER_COPY:/opt/node-sass -w /opt/node-sass mhart/alpine-node:$TRAVIS_NODE_VERSION /bin/sh -c "apk add --no-cache python=2.7.12-r0 make gcc g++ && npm install --unsafe-perm --verbose && npm test";
```

Comment:
Done. Must have left it while debugging. I'll probably move it over to a Dockerfile if I get this running properly